src/plugins/footnotes/index.ts

```diff
@@ -203,13 +214,32 @@ export default () => {
                 )
               )
             }
+
+            if (
+              tableInlineFootnoteIds &&
+              !tableInlineFootnoteIds.has(node.attrs.id)
+            ) {
+              decorations.push(
+                Decoration.widget(
+                  pos + node.nodeSize - 1,
```

Comment:
How does that `node.nodeSize` work? It indicates the footnote node size, right? So to me, it doesn't look like a correct calculation.

Reply:
yes, it's working for me!